lib/tag-finder.coffee

```diff
@@ -132,7 +132,7 @@ class TagFinder
   #
   # Returns a string with the name of the most recent unclosed tag.
   tagsNotClosedInFragment: (fragment) ->
-    @parseFragment fragment, [], /<(\w[-\w]*)|<\/(\w[-\w]*)/, -> true
+    @parseFragment fragment, [], /<(\w[-\w]*(?:\:\w[-\w]*)?)|<\/(\w[-\w]*(?:\:\w[-\w]*)?)/, -> true
```

Comment:
From reading a Wikipedia article on namespaces it appears that URIs can also be used as namespaces, in which case this regex will need to be expanded...a lot.